<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login_condition</class>
 <widget class="QDialog" name="login_condition">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>220</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>200</width>
     <height>501</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>200</width>
      <height>65</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>201</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>我在线上</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>201</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Q我吧</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>75</y>
      <width>200</width>
      <height>101</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>5</y>
       <width>201</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>离开</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>35</y>
       <width>201</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>忙碌</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>65</y>
       <width>201</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>请勿打扰</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>176</y>
      <width>200</width>
      <height>72</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>5</y>
       <width>201</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>隐身</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>35</y>
       <width>201</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>离线</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>248</y>
      <width>201</width>
      <height>38</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>5</y>
       <width>201</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>添加状态信息</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>285</y>
      <width>201</width>
      <height>72</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>5</y>
       <width>201</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>关闭所有声音</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_10">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>35</y>
       <width>201</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>关闭头像闪动</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>355</y>
      <width>201</width>
      <height>43</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButton_11">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>8</y>
       <width>201</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>锁定QQ  Ctrl+Alt+L</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_8">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>398</y>
      <width>201</width>
      <height>101</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButton_12">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>5</y>
       <width>201</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>系统设置</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_13">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>35</y>
       <width>201</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>我的资料</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_14">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>65</y>
       <width>201</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>我的QQ中心</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
